parts/linux/cloud-init/artifacts/components.json

```diff
@@ -36,6 +36,22 @@
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-cni:*",
       "amd64OnlyVersions": [],
       "multiArchVersionsV2": [
+        {
+          "renovateTag": "registry=https://mcr.microsoft.com, name=containernetworking/azure-cni",
+          "latestVersion": "v1.4.56",
+          "containerImagePrefetch": {
+            "latestVersion": {
+              "binaries": [
+                "/dropgz"
+              ]
+            },
+            "previousLatestVersion": {
```

Comment:
need to remove this if no previousLatest is present

Reply:
`generating container prefetch script content: container image "mcr.microsoft.com/containernetworking/azure-cni:*" specifies a previousLatestVersion prefetch optimization, but does not have a previousLatestVersion`

Reply:
oops, fixed